.github/labeler.yml

```diff
@@ -73,6 +73,10 @@ comp:tril:
   - fvtest/compilertriltest/**/*
   - fvtest/tril/**/*
 
+comp:utilities:
```

Comment:
I didn't realize we had a `comp:utilities`.

I think `include_core` should map to `comp:core`

Reply:
Fixed in force push.